LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Maximilian Schiller
+Copyright (c) Her Majesty the Queen in Right of Canada, as represented by the Minister of ESDC, 2024
```

Comment:
We should keep the original copyright notice and add ours above with the start date. The work prior to now has been done by the someone else but the project not being maintained forced us to fork and start maintaining a derivative version.

Reply:
I would also write the name of the department out long
"Employment and Social Development Canada"

Reply:
Ok, I changed that to CDS, but still not sure if it is right since CDS does not have a specified minister (ie it is a minister of broader department not just CDS), but maybe that is ok? Anyways, let me know if I got it right now and THANK YOU so much for all the guidance.